lib/components/viewers/trip-viewer.js

```diff
@@ -38,6 +40,16 @@ const HeaderText = styled.h1`
   margin: 2px 0 0 0;
 `
 
+const TripDescription = styled.div`
+  border-left: 4px solid #13c1c1;
```

Comment:
Hmm I tried a different approach, without the border, tightening the text block and adding a subtle background color. I'm not sure it's quite there, but maybe it's in the right direction? Let me know!

Reply:
This is it! I love it. My only 2 comments on it at this point are to match the color with the highlights on the blobs (I'd adjust the blob highlight color -- I like yours more)

and maybe adding
```css
border-radius: 5px;
box-shadow: 2px 2px 5px 1px rgba(0, 0, 0, 0.02);
```
to the box? What do you think

Reply:
LOVE this! I added these, and a TINY bit of padding, and I think it's looking pretty good!

Changing the highlight color on the blobs to match the description is tricky because it's unfortunately barely visible. The description background is decoration, but the highlight is at least in part trying to communicate information? I split the difference and made the opacity `0.5` but it may just look inconsistent, so I'm very open to feedback!